app/validators/postcode_validator.rb

```diff
@@ -0,0 +1,7 @@
+class PostcodeValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    unless PostcodeApi.new(value).postcode_valid?
```

Comment:
I would suggest that before we hit the API, we validate the format first. Again it's super easy using the `uk_postcode` gem (for example). But if I put an `bla bla` postcode into the input, the API shouldn't be hit. Both for performance and rate limit reasons.

Reply:
Good idea